src/button/IconButton.tsx

```diff
@@ -8,11 +8,15 @@ import { forwardRefWithAs } from "../utils/types";
 
 export type IconButtonProps = ButtonProps & {
```

Comment:
Hmmm 🤔 we can change this too 

```ts
export type IconButtonProps = ReakitButtonProps &
  Pick<ButtonProps, "size" | "variant">;
```

Considering we change the comments to be more specific like: "How large should the component be?"